AsyncDisplayKit/ASTableView.mm

```diff
@@ -800,8 +806,6 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
     [_rangeController configureContentView:cell.contentView forCellNode:node];
 
     cell.node = node;
-    cell.backgroundColor = node.backgroundColor;
-    cell.selectionStyle = node.selectionStyle;
 
     // the following ensures that we clip the entire cell to it's bounds if node.clipsToBounds is set (the default)
     // This is actually a workaround for a bug we are seeing in some rare cases (selected background view
```

Comment:
@SaifAlDilaimi :) It looks like you forgot one more — .clipsToBounds

Since it's also in setNode in the Collection case, let's make sure the Table matches.  You can actually delete this comment while moving the one line of code, as the comment references an old bug that has since been resolved.

I don't see any other changes to request (this is part of the last request), and so once @maicki confirms, I think we should land this.  It would be great to have tests; perhaps we can track that as a separate task (since I know you were expecting this to be a very quick project)?